apps/calendar/src/helpers/css.ts

```diff
@@ -37,3 +41,22 @@ export function toPercent(value: number) {
 export function toPx(value: number) {
   return `${value}px`;
 }
+
+// eslint-disable-next-line complexity
+export function getEventItemColors(uiModel: EventUIModel, calendarColor: CalendarColor) {
+  const eventColors = uiModel.model.getColors();
+
+  return {
+    color: eventColors.color ?? calendarColor.color ?? DEFAULT_EVENT_COLORS.color,
+    borderColor:
+      eventColors.borderColor ?? calendarColor.borderColor ?? DEFAULT_EVENT_COLORS.borderColor,
+    backgroundColor:
+      eventColors.backgroundColor ??
+      calendarColor.backgroundColor ??
+      DEFAULT_EVENT_COLORS.backgroundColor,
+    dragBackgroundColor:
+      eventColors.dragBackgroundColor ??
+      calendarColor.dragBackgroundColor ??
+      DEFAULT_EVENT_COLORS.dragBackgroundColor,
+  };
```

Comment:
```suggestion
  return {
    ...DEFAULT_EVENT_COLORS,
    ...calendarColor,
    ...eventColors
  };
```
같은 방식은 안되려나요.. 난해하네요

Reply:
아이디어는 괜찮은데 이벤트에서 컬러 꺼내올 때가 문제라서 저거 그대로는 적용하기 어려울거같네요.

https://github.com/nhn/tui.calendar/pull/1136/files#diff-74d5d037a374bae6796be806d361609214767e982ecf1780f60d75acd0e6b7d6R332-R339

만약에 할당된 컬러 속성값이 없으면 `{ color: undefined }` 이런식으로 리턴될텐데, key는 명백하게 정의가 되어있으므로 최종적으로 `undefined` 로 값이 덮어씌워져 버릴거에요. 적절한 도구를 사용해서 깔끔하게 머지시키면 좋긴 하겠어요.